README.md

```diff
@@ -62,7 +66,9 @@ stored in `conf/options.php`, which HotCRP marks as world-unreadable. You must
 ensure that your PHP can read this file.
 
 2. Edit `conf/options.php`, which is annotated to guide you.
-(`lib/createdb.sh` creates this file based on `src/distoptions.php`.)
+(`lib/createdb.sh` creates this file based on `src/distoptions.php`.) Make
+sure that `conf/options.php` is readable by the user that your webserver runs
+as.
```

Comment:
Ah! In my case, this code did not trigger because Apache was not yet running (I had installed it, but not started the service), although the warning on line 503 should have.

I may have overlooked it, I guess -- so never mind, feel free to drop this.